x-pack/plugins/observability/server/assets/constants.ts

```diff
@@ -5,6 +5,7 @@
  * 2.0.
  */
 
+export const SLO_MODEL_VERSION = 2;
 export const SLO_RESOURCES_VERSION = 2;
 export const SLO_SUMMARY_TRANSFORMS_VERSION = 3;
```

Comment:
@simianhacker Let's sync with the value we want to use. So we don't have different version to keep track

Reply:
Good question, whatever value we use, any time that changes, it will trigger the need to `reset` the SLO definitions. What's the harm in leaving it "as is" other than the complexity of having three values?

What if we added a comment to above `SLO_MODEL_VERSION` that says:

> Anytime we increment `SLO_MODEL_VERSION`, this will notify the user to "RESET" their SLO definitions which in turn re-indexes the SLI values and SLO summaries.  Be judicious about changing this value.